jujugui/static/gui/src/test/test_viewlet_manager.js

```diff
@@ -100,6 +100,9 @@ describe('Viewlet Manager', function() {
     });
   });
 
+  // Required so that the cleanups instance can be attached.
+  beforeEach(function() {});
```

Comment:
So is this a test-by-test situation, or do all test cases now need before/afterEach functions?